main.py

```diff
@@ -1,12 +1,142 @@
 import requests
+from video_store import VideoStore
 
 URL = "http://127.0.0.1:5000"
 BACKUP_URL = "https://retro-video-store-api.herokuapp.com"
 
+
 def main():
+    videos = VideoStore()
     print("WELCOME TO RETRO VIDEO STORE")
-    pass
+    print("\n************************************\n")
+    print("Please choose an option")
+    print("Enter 1 to manage videos")
+    print("Enter 2 to customers")
+    print("Enter 3 to check-in/check-out")
+
+    main_choice = input("Enter your choice: ")
+    print("\n************************************\n")
+
+    if main_choice == "1":
+        print("What would you like to do in videos? ")
+        print("Enter 1 to see all videos")
+        print("Enter 2 to add video")
+        print("Enter 3 to update a video")
+        print("Enter 4 to delete a video")
+        print("Enter 5 to get a video")
+        choice = input("Enter your choice: ")
+        if choice == "1":
+            print("Here is a list of all videos")
+            for video in videos.get_all_videos():
+                print(video)
+
+        elif choice == "2":
+            print("You are about to add a new movie.")
+            title = input("What is the title of the movie? ")
+            release_date = input(f"When was {title} released? Please enter in yyyy-mm-dd. ")
+            total_inventory = input(f"What is the total inventory of {title}? ")
+            response = videos.add_video(title = title, release_date = release_date, total_inventory = total_inventory)
+
+            print(f"{title} was created")
+
+        elif choice == "3":
+            video_id = input("Enter a video id: ")
+            print(f"You are about to update a movie with video id: {video_id}")
+            title = input("What is the new movie title? ")
+            release_date = input(f"When was {title} released? Please enter in yyyy-mm-dd. ")
+            total_inventory = input(f"What is the total inventory of {title}? ")
+            response = videos.edit_video(video_id = video_id, title = title, release_date = release_date, total_inventory = total_inventory)
+
+            print(f"{title} is now updated")
+
+        elif choice == "4":
+            try:
+                print("You are about to delete a movie.")
+                video_id = input("Enter a video id: ")
+                videos.delete_video(video_id)
+                print("This video has been deleted.")
+            
+            except:
```

Comment:
Unless there's some other error in the system, inputing an invalid video id will not trigger this except sequence.  The request will return an error code, but that doesn't throw an exception.  A secondary issue is what is the desired behavior if a user inputs a video id that does not exist multiple times?  If the except block did catch an invalid id, inputing another invalid id inside the except block would not catch the error.  Using a while loop with some kind of error checking makes it possible to continue to request the information from the user until they input a valid id.